packages/tools/release-utils-internal/src/git.ts

```diff
@@ -39,3 +42,37 @@ export async function addRemoteWithAuthentication(repoUrl: string, remoteName: s
 
     await execShellCommand(`git remote add "${remoteName}" "${getGHRepoAuthUrl(repoUrl)}"`);
 }
+
+export function noChanges(): boolean {
+    return /nothing to commit/i.test(sh.exec("git commit --dry-run -uno", { silent: true }));
+}
+
+export function commit(msg: string): void {
+    if (commitMessageRegEx.test(msg)) {
+        sh.exec(`git commit -m ${msg}`);
+    } else {
+        throw new Error("Invalid commit message: Please use quotest for commit message");
```

Comment:
Typo. And why would we enforce this?

Reply:
If I understand correctly it enforces user on this message pass already quoted string, why not to put quotes ourselves instead of making user to write `commit('"My message"')`?